docs/recipes/precompiling-with-webpack.md

```diff
@@ -29,3 +37,226 @@ module.exports = {
 The important bits are `target: 'node'`, which ignores Node.js-specific `require`s (e.g. `fs`, `path`, etc.) and `externals: nodeModules` which prevents webpack from trying to bundle external Node.js modules which it may choke on.
 
 You can now run `$ ava _build/tests.js` to run the tests contained in this output.
+
+### Multiple test files
+Things are a little more complicated with multiple test files. We recommend [using babel-register](babelrc.md) until the performance penalty becomes too great.
+
+The possible approaches are:
+
+- [Refer precompiled source in tests](#refer-precompiled-source-in-tests)
+- [Single entry file](#single-entry-file)
+- [Multiple entry files](#multiple-entry-files)
+- [Multiple entry files with externalized source files](#multiple-entry-files-with-externalized-source-files)
+
+#### Refer precompiled source in tests
+Source files can be compiled with source maps(for coverage) to `_src` folder and referenced in tests. While this is less than elegant, it performs well and the worklow can be optimized with [webpack's watch mode](https://webpack.js.org/configuration/watch/).
```

Comment:
> Why `_src` here versus `_build` above?

I was thinking `_build` for tests and `_src` for source. 

> How would users do this precompilation?

I was thinking `babel-cli`.. I'll change this to mention [`babel-cli` watch mode](https://babeljs.io/docs/usage/cli/#babel) instead of webpack. Sound good?